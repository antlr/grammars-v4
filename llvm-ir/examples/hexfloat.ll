@a = global half 0xH4400
@b = global half 0xH2E66
