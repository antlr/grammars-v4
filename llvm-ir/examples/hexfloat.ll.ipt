compilationUnit
 topLevelEntity
  globalDef
   @a
   =
   immutable
    global
   type
    floatType
     floatKind
      half
   constant
    floatConst
     0xH4400
 topLevelEntity
  globalDef
   @b
   =
   immutable
    global
   type
    floatType
     floatKind
      half
   constant
    floatConst
     0xH2E66
 <EOF>
