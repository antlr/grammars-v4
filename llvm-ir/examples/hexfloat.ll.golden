@a = global half 4.0
@b = global half 0xH2E66
